.github/ISSUE_TEMPLATE/suggest_wallet.md

```diff
@@ -6,98 +6,192 @@ labels: "wallet :purse:,"
 
 **Please describe the measures taken to ensure the wallet's security and provide documentation wherever possible**
 
-<!-- Please provide a link to a report or repo. If you haven't been audited but think your wallet should be listed anyway, explain here. -->
+<!-- Please provide a link to any security audits or code reports. If you haven't been audited but think your wallet should be listed anyway, explain here. -->
 
-**Does the wallet support layer 2 networks?**
+**Spam protection?**
 
-<!-- Please list which L2 solutions the wallet supports. -->
+<!-- Does the wallet employ any practices to warn users against potential spam (e.g. when interacting with suspicious accounts/contracts)? -->
 
-**Can the wallet be used with arbitrary Ethereum RPC endpoint?**
+### Features
 
-<!-- Does wallet allow user to use the wallet with own Eth client? -->
+**Does the wallet have hardware wallet support?**
 
-**Does the wallet have fiat on-ramps?**
+<!-- Please provide information on how a user can connect a hardware wallet to this wallet. -->
 
-<!-- Does the wallet subsidise transaction fees for all or some wallet activity? -->
+**Does the wallet support WalletConnect?**
 
-**Does the wallet allow users to explore dapps?**
+<!-- Please provide information on if the wallet supports WalletConnect. -->
 
-<!-- Is the wallet integrated with WalletConnect? Does it have a dapp browser? Or can users connect to a dapp using the wallet?  -->
+**Does the wallet support importing Ethereum RPC endpoints?**
 
-**Does the wallet have integrated defi/financial tools?**
+<!-- Please provide documentation on how a user can import an Ethereum RPC into the wallet. -->
 
-<!-- Can users borrow/earn/lend assets directly from a screen in the wallet?  -->
+**Does the wallet support viewing and interacting with NFTs?**
 
-**Can a user withdraw to their card?**
+<!-- Please provide information on the experience of NFTs in the wallet. -->
 
-<!-- Can a user cash out their crypto assets directly to a card or bank account? -->
+**Does the wallet support connecting to Ethereum applications?**
 
-**Does the wallet offer limits protection?**
+<!-- Please provide documentation for how users connect to applications. List examples (ie. connect wallet to dapp, in wallet browser, etc.) -->
 
-<!-- Can a user set a limit for transfers to protect their assets? -->
+**Does the wallet support staking directly?**
 
-**Does the wallet allow high-volume purchases?**
+<!-- Please provide documentation on direct staking this wallet supports. -->
 
-<!-- Can a user buy more than $2000 worth of crypto in one go? -->
+**Does the wallet support swaps directly?**
 
-**Does the wallet have an integrated token swap?**
+<!-- Please provide documentation on swaps. -->
 
-<!-- Can a user swap ETH for other tokens from within a screen in the wallet? -->
+**Does the wallet support multi-chain networks?**
```

Comment:
```suggestion
**Does the wallet support layer 2 networks?**
```
Curious what folks think of "layer 2" vs. "multi-chain"? Or should we include both?

Ultimately I suspect L2 is what we care about. Users may want other EVM chains. There could also be user benefit in e.g. Bitcoin support in a wallet (like Ledger does).

Reply:
Yeah, I see L2 and multi-chain as two different feature sets... one is contained within the Ethereum security bubble, the other is many different security bubbles.

Yes, in general, it's about which networks the wallet can connect to... but someone who uses Ethereum may reasonably want to find a wallet that supports L2s so they can inherit Ethereum's L1 security and still afford gas.

Meanwhile, someone who uses many cryptocurrencies on many chains care that their wallet also supports these other chains.

So my vote would be to include both. We should ask what non-EVM chains are compatible, and also what Ethereum L2s are compatible, imo.

Reply:
I agree with the notion here, but my pushback would be from our discussions of what layer 2's are. I think a user would want to look at validiums or optimistic chains as well, which I would say still fall under the bundle of scaling Ethereum, but we don't classify it as a layer 2. That being said, I think that is more captured in the RPC importing question. Down for a discussion, but I think I agree with this suggestion.